tuf/tuf_test.go

```diff
@@ -863,16 +884,29 @@ func TestRemoveExistingAndNonexistingTargets(t *testing.T) {
 	// still no metadata
 	_, ok = repo.Targets["targets/test"]
 	require.False(t, ok)
-}
 
-// Removing targets from a role that exists but without metadata succeeds.
-func TestRemoveTargetsNonexistentMetadata(t *testing.T) {
```

Comment:
Note - this test didn't actually seem to do what it claimed - the body was a duplicate of `TestRemoveTargetsRoleDoesntExist`.  Since the previous test was supposed to test adding existing and non-existing targets anyway, I just removed this test and updated the previous one.